tutorials/cloud-iot-rtdp/bin/create_cert.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+openssl req -x509 -newkey rsa:2048 -keyout rsa_private.pem -nodes -out rsa_cert.pem -subj "/CN=unused"
```

Comment:
Default expiration on these certs is 30 days, you may want to extend it by passing a param like `-days 1000` after the `keyout` argument